packages/documents/graphql/schema-types.js

```diff
@@ -95,6 +95,21 @@ type Meta {
   indicateChart: Boolean
   indicateGallery: Boolean
   indicateVideo: Boolean
+  # name of a special discussion type
+  discussionType: String
+  # tag properties like colors or title texts
+  tagMappings: [TagMapping!]
+}
+
+type TagMapping {
+  tag: String!
+  text: String!
+  color: TagMappingColor
+}
+
+type TagMappingColor {
+  light: String!
+  dark: String!
```

Comment:
IMO we should remove this here again. I can get it from `content.meta` without it being in the schema. If we end up using it again we should move it to the discussion type.

Reply:
Reverted.